lib/valkyrie/persistence/fedora/persister/model_converter.rb

```diff
@@ -23,7 +23,7 @@ def convert
       end
 
       def properties
-        resource_attributes.keys - [:new_record]
+        resource_attributes.keys - [:new_record, :last_modified]
```

Comment:
Can you comment why this is needed (to supplement the token)

Reply:
Also should this be added to the reserved attributes list? https://github.com/samvera-labs/valkyrie/blob/dc4043b80f938f7ebd1493f4528e85275df7529d/lib/valkyrie/resource.rb#L44

Reply:
I'm torn about whether `last_modified` should be a reserved attribute.  It might be a cleaner implementation to do it that way, but it would also spread this outside of the Fedora persister.